exoatlas/populations/exoplanets/exoplanets.py

```diff
@@ -983,3 +983,8 @@ def update_reference(self, references, keys=None, planets=None, verbose=False):
 
         if nothing_happened:
             warnings.warn(warning_message)
+
```

Comment:
@MiriCaradonna , by importing your functions directly into the class definition for the `Exoplanets` population object, now we should be able to call your function directly as a method of the population, like this:
```python
from exoatlas import *
e = TransitingExoplanets()
e.find_all_observable_transits(observer=..., obs_start_time=...)
``` 
as opposed to having to pass the population explicitly into a standalone function like 
```python
from exoatlas import *
e = TransitingExoplanets()
find_all_observable_transits(population=e, observer=..., obs_start_time=...)
```
There's no intrinsic reason to prefer one over the other! For starting to dip our toes into using classes, this effectively works because the `population` variable you're using in your functions is effectively (literally!) serving as the `self` argument that often gets passed into class method definitions.